src/parse_logcfg.c

```diff
@@ -110,20 +109,32 @@ static bool isCommentLine(char *buffer) {
 
 int parse_configfile(FILE *fp) {
     int status = PARSE_OK;
-    char buffer[2000];
+    char *buffer;
```

Comment:
`buffer` has to be initialized to NULL. Please also check other occurrences (e.g. checklogfile.c)

Reply:
Thanks. Sorry recently I suffer for lack of free time. so didn't have much time to finish this change.

Reply:
No problem, just take your time.